lib/cryptoexchange/exchanges/therocktrading/services/market.rb

```diff
@@ -40,7 +46,7 @@ def adapt(output, market_pair)
           ticker.last = NumericHelper.to_d(market['last'])
           ticker.high = NumericHelper.to_d(market['high'])
           ticker.low = NumericHelper.to_d(market['low'])
-          ticker.volume = NumericHelper.to_d(HashHelper.dig(market, 'volume_traded', 'amount'))
+          ticker.volume = NumericHelper.to_d(market['volume'])
```

Comment:
can double check? volume in base currency should be `volume_traded` right?
```
date | "2019-02-19T07:10:58.196Z"
-- | --
fund_id | "BTCEUR"
bid | 3433.06
ask | 3440.16
last | 3440.16
open | 3209.03
close | 3414.48
low | 3199.15
high | 3453.73
volume | 647959.20848
volume_traded | 190.8481
```

if your base is BTC, target is EUR, volume should be volume_Traded? unless your base/target is reversed?